.golangci.yml

```diff
@@ -0,0 +1,25 @@
+linters:
+  auto-fix: false
+  enable:
+    - errcheck
+    - goimports
+    - golint
+    - gosec
+    - misspell
+    - scopelint
+    - unconvert
+    - unparam
+    - interfacer
+    - nakedret
+    - gocyclo
+    - dupl
+    - goconst
+    - lll
+run:
+linters-settings:
+  errcheck:
+    check-type-assertions: true
+  lll:
+    line-length: 250
```

Comment:
We recently outright removed `lll` in the SDK linting. But 250 seems like a completely reasonable limit to catch something egregious.